Kconfig.zephyr

```diff
@@ -306,6 +306,13 @@ config BUILD_OUTPUT_STRIPPED
 	  Build a stripped binary. This will build a zephyr.stripped file need
 	  by some platforms.
 
+config BUILD_BOARD_IMAGE
+	bool "Build a board-specific image"
+	help
+	  Build a board specific binary. This may differ from board to board
```

Comment:
Document that a script is expected in the board directory under support

Reply:
Done.